server/konnectors/edf.coffee

```diff
@@ -687,12 +734,12 @@ fetchEdeliaToken = (requiredFields, entries, data, callback) ->
     K.logger.info "fetchEdeliaToken"
     request.post 'https://api.edelia.fr/authorization-server/oauth/token',
         form:
-            client_id: requiredFields.edeliaClientId
+            client_id:
+                'sha1pae0Pahngee6uwiphooDie7thaiquahf2xohd6IeFeiphi9ziu0uw3am'
```

Comment:
Until now, I was waiting for a dedicated clientId. The previous field hack allowed me to test the konnector with a clientId of another app.
Definitely, it's not a data handled by the user.